src/WebJobs.Script/Host/DebugManager.cs

```diff
@@ -16,11 +16,11 @@ internal class DebugManager : IDebugManager
         private readonly string _rootLogPath;
         private readonly ILogger _logger;
 
-        public DebugManager(IOptions<ScriptJobHostOptions> scriptOptions, IDebugStateProvider debugStateProvider,
+        public DebugManager(IOptions<ScriptApplicationHostOptions> scriptOptions, IDebugStateProvider debugStateProvider,
```

Comment:
This needs to use an `IOptionsMonitor` here now, and make sure we're not caching the value.